### ISSUE TITLE
Support `jvm_opts` on scala_toolchain

### ISSUE THREAD

**Issue by arjantop** [bazelbuild/rules_scala#706]
Add support for `jvm_opts`, equivalent to java_toolchain: https://docs.bazel.build/versions/master/be/java.html#java_toolchain.jvm_opts

**Comment by stephenjudkins** [comment]
I believe (though I am not sure!) that this can be achieved through `--jvm_flag` either passed at the command line or in a `.bazelrc` file

**Comment by libsamek** [comment]
Yeah, there is a --jvmopt, which has no effect in combination with scala_binary and scala_test.
